superset-frontend/src/dashboard/components/nativeFilters/FilterConfigModal.tsx

```diff
@@ -0,0 +1,283 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { findLastIndex, uniq } from 'lodash';
+import shortid from 'shortid';
+import { DeleteFilled } from '@ant-design/icons';
+import { styled, t } from '@superset-ui/core';
+import { Form } from 'src/common/components';
+import { StyledModal } from 'src/common/components/Modal';
+import { LineEditableTabs } from 'src/common/components/Tabs';
+import { DASHBOARD_ROOT_ID } from 'src/dashboard/util/constants';
+import { usePrevious } from 'src/common/hooks/usePrevious';
+import ErrorBoundary from 'src/components/ErrorBoundary';
+import { useFilterConfigMap, useFilterConfiguration } from './state';
+import FilterConfigForm from './FilterConfigForm';
+import { FilterConfiguration, NativeFiltersForm } from './types';
+
+const StyledModalBody = styled.div`
+  display: flex;
+  flex-direction: row;
+  .filters-list {
+    width: 200px;
```

Comment:
```suggestion
    width: ${({ theme }) => theme.gridUnit * 50}px;
```
Not sure if this is also `// arbitrary` comment worthy or not, but might as well stick to the grid :D